components/airtable/airtable.app.mjs

```diff
@@ -40,7 +40,7 @@ export default {
     viewId: {
       type: "string",
       label: "View",
-      description: "The view ID. If referencing a **Table** dynamically using data from another step (e.g., `{{steps.mydata.$return_value}}`), automatic view options won't work when configuring this step. Please enter a custom expression to specify the **View**.",
+      description: "The view ID. If referencing a **Table** dynamically using data from another step (e.g., {{steps.trigger.event.metadata.tableId}}), you will not be able to select from the list of Views for this step. Please enter a custom expression to specify the **View**.",
```

Comment:
nit, but is there a reason you removed the code formatting?

Reply:
Same as above